### ISSUE TITLE
Sprint 2 Review

### ISSUE THREAD

**Issue by julianredmond** [kittens-mittens/team-app#13]
So far we have done a lot of product research including competitive analyses and personas. We have come up with a app name, a logo and a visual design guide. We are currently completing the wireframes and have filled out the UX rubric.

How did you & the sprint succeed? What was positive?

The sprint kept our tasks organized and allowed us to know which team member needed to work on which tasks. The sprint was successful in ensuring that our team did not forget to complete any tasks.

How did you & the sprint fail? What was negative?

Everything went smoothly with this sprint and we didn't encounter any negativity.

@thomasjbradley

**Comment by lees3** [comment]
I agree. Sprint has been a nice visual aid to remind us of all of the tasks that need to be completed. It can also demonstrate if any team members appear to be behind on their tasks, and if they may need help with their task(s).

**Comment by JasmineCT** [comment]
How did you & the sprint succeed? What was positive?
The sprint helped to keep things organized and let us know what need to get done for each task. eg. Competitive analysis, individually contributed to it and what need to get done, but on person submits. Overall helped to see what need to be done.

How did you & the sprint fail? What was negative?
I also agree nothing really failed with the sprint in views of organization.

**Comment by thomasjbradley** [comment]
💥
